.github/workflows/test_examples.yml

```diff
@@ -14,7 +14,7 @@ on:
       - devenv.yaml
 
 env:
-  DEVENV_VERSION: v0.6.3
+  DEVENV_VERSION: v1.0.3
```

Comment:
Do we want to test the examples / our configuration with both, `v1.0.3` and `v0.6.3`, or is sufficient to use the latest version?

Reply:
Latest suffices 🙂